snapshot/lib/snapshot/options.rb

```diff
@@ -137,6 +137,12 @@ def self.available_options
                                      description: "A list of videos that should be added to the simulator before running the application",
                                      type: Array,
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :html_template,
+                                     env_name: 'SNAPSHOT_HTML_TEMPLATE',
+                                     short_option: "-e",
+                                     description: "A path to screenshots.html template",
+                                     default_value: false,
```

Comment:
Would it maybe make sense to set the default to `File.join(Snapshot::ROOT, "lib", "snapshot/page.html.erb")`? Then you could simplify the code a bit and also clean up the naming of the variables a bit more.